crates/kv/src/implementors/redis.rs

```diff
@@ -13,18 +13,23 @@ use slight_runtime_configs::get_from_state;
 #[derive(Debug, Clone)]
 pub struct RedisImplementor {
     client: Client,
+    container_name: String,
 }
 
 impl RedisImplementor {
-    pub fn new(slight_state: &BasicState, _name: &str) -> Self {
+    pub fn new(slight_state: &BasicState, name: &str) -> Self {
         let connection_string = get_from_state("REDIS_ADDRESS", slight_state).unwrap();
         let client = redis::Client::open(connection_string).unwrap();
-        Self { client }
+        let container_name = name.to_string();
+        Self {
+            client,
+            container_name,
+        }
     }
 
     pub fn get(&self, key: &str) -> Result<Vec<u8>> {
         let mut con = self.client.get_connection()?;
-        let val: Vec<u8> = con.get(key)?;
+        let val: Vec<u8> = con.get(format!("{}:{}", self.container_name, key))?;
```

Comment:
why?

Reply:
we want `folder_name:key` format to differentiate same keys from different named entities.

Reply:
Do we throw an error if the `folder_name` is equal?

Reply:
I didn't understand your question. If folder_name is the same, then we are performing `get`, `set` and `delete` on that folder. I just don't want the following to happen:

```rust
let kv1 = Kv::open("container-a");
let kv2 = Kv::open("container-b");
assert!(kv1.get("key") == kv2.get("key"));
```

Reply:
I was just thinking of the use case where we do:
```rs
let kv1 = Kv::open("container-a");
let kv2 = Kv::open("container-a");
```

Reply:
Ah, so in the above example, if `kv1` writes a key, then `kv2` will see it.